R/s3_methods.R

```diff
@@ -229,17 +229,55 @@ dimnames.RPolarsLazyFrame = function(x) list(NULL, names(x))
 
 #' Convert to a data.frame
 #'
-#' @param x A [DataFrame][DataFrame_class] or [LazyFrame][LazyFrame_class]
-#' @param ... Any arguments passed to [data.frame()].
-#'
+#' Equivalent to `as_polars_df(x, ...)$to_data_frame(...)`.
+#' @param x A object to convert to a [data.frame].
+#' @param ... Additional arguments passed to methods.
+#' @inheritParams DataFrame_to_data_frame
+#' @seealso
+#' - [as_polars_df()]
+#' - [`<DataFrame>$to_data_frame()`][DataFrame_to_data_frame]
 #' @export
 #' @rdname S3_as.data.frame
-as.data.frame.RPolarsDataFrame = function(x, ...) x$to_data_frame(...)
+as.data.frame.RPolarsDataFrame = function(x, ..., int64_conversion = polars_options()$int64_conversion) {
+  x$to_data_frame(..., int64_conversion = int64_conversion)
+}
 
 
 #' @export
+#' @inheritParams as_polars_df.RPolarsLazyFrame
 #' @rdname S3_as.data.frame
-as.data.frame.RPolarsLazyFrame = function(x, ...) x$collect()$to_data_frame(...)
+as.data.frame.RPolarsLazyFrame = function(
+    x,
+    ...,
+    n_rows = Inf,
+    type_coercion = TRUE,
+    predicate_pushdown = TRUE,
+    projection_pushdown = TRUE,
+    simplify_expression = TRUE,
+    slice_pushdown = TRUE,
+    comm_subplan_elim = TRUE,
+    comm_subexpr_elim = TRUE,
+    streaming = FALSE,
+    no_optimization = FALSE,
+    inherit_optimization = FALSE,
+    collect_in_background = FALSE) {
+  as_polars_df.RPolarsLazyFrame(
```

Comment:
```suggestion
  as_polars_df(
```
Not necessary since the class of `x` has to be `RPolarsLazyFrame`

Reply:
I don't understand your argument. If the arg `x` doesn't have the class `RPolarsLazyFrame` then calling `as.data.frame()` on it won't dispatch it to `as.data.frame.RPolarsLazyFrame()`. So putting `.RPolarsLazyFrame` inside the function is redundant. This is not very important though and I already approved the PR

Reply:
Indeed, you are right about this place, it may be redundant since there is no possibility that anything other than RPolarsLazyFrame will be entered.